src/pages/AnimeFenix/main.ts

```diff
@@ -0,0 +1,67 @@
+import { pageInterface } from '../pageInterface';
+
+export const AnimeFenix: pageInterface = {
+  name: 'AnimeFenix',
+  domain: 'https://animefenix.tv',
+  languages: ['Spanish'],
+  type: 'anime',
+  isSyncPage: url => {
+    const path = url.split('/')[3];
+    return path === 'ver';
+  },
+  sync: {
+    getTitle: url => {
+      const titleElement = document.querySelector('.hero h1');
+      let title = titleElement?.textContent ? titleElement.textContent.trim() : null;
+    
+      if (!title) {
+        const urlTitle = url.split('/')[4];
+        title = urlTitle ? urlTitle.replace(/-/g, ' ').replace(/\b\w/g, l => l.toUpperCase()) : '';
+      }
+      const titleParts = title.split(' ');
+      if (titleParts.length > 1 && !Number.isNaN(Number(titleParts[titleParts.length - 1]))) {
+        titleParts.pop();
+        title = titleParts.join(' ');
+      }
+      return title;
+    },    
+    getIdentifier: url => {
+      return AnimeFenix.sync.getTitle(url);
+    },
+    getOverviewUrl: url => {
+      const urlParts = url.split('/');
+      if (urlParts[3] === 'ver') {
+        urlParts.splice(3, 1);
+      }
+      urlParts[urlParts.length - 1] = urlParts[urlParts.length - 1].replace(/-\d+$/, '');
+      return urlParts.join('/');
+    },    
+    getEpisode: url => {
```

Comment:
Episode wrong: 
https://www3.animefenix.tv/ver/kono-yo-no-hate-de-koi-wo-utau-shoujo-yu-no-26.5
https://www3.animefenix.tv/ver/kusuriya-no-hitorigoto-22?page=232323

Reply:
i need to let the 26.5 or is correct if is 26?

Reply:
@lolamtisch